nntrainer/graph/network_graph.cpp

```diff
@@ -96,14 +97,7 @@ LayerNode &NetworkGraph::getLayerNode(unsigned int ith) {
 }
 
 LayerNode &NetworkGraph::getSortedLayerNode(unsigned int ith) {
-
-  for (unsigned int i = 0; i < Sorted.size(); ++i) {
-    if (Sorted[i].index == ith) {
-      return Sorted[i];
-    }
-  }
-
-  throw std::invalid_argument("Cannot find Layer");
+  return Sorted[ith];
```

Comment:
If Sorted.empty() throw?

Reply:
I thought c++ used to throw in this situation, but it does not. Added.

Reply:
Awesome